### ISSUE TITLE
Option to modify instances of recurring reminders

### ISSUE THREAD

**Issue by clarfon** [scottleedavis/mattermost-plugin-remind#133]
We use this plugin to remind people about meetings, and it's been very useful for this!

Unfortunately, there's no way to "cancel" a meeting, i.e. mute a reminder for a set period of time. This would be very nice to have.

**Comment by scottleedavis** [comment]
I think this idea is appealing, I often want to do the same thing.  The _how_ of making this happens would need to be flushed out.  Perhaps a button in `/remind list` 'mute' on recurring reminders?  Thoughts?

**Comment by clarfon** [comment]
Yeah, it makes the most sense to me to have this be an option in `/remind list`, at least for a first implementation.

**Comment by scottleedavis** [comment]
@esethna how do you feel about this request?
